src/build/env.py

```diff
@@ -82,10 +90,13 @@ def _subprocess(cmd: List[str]) -> None:
 
 
 class IsolatedEnvBuilder:
-    """Builder object for isolated environments."""
+    """Create and dispose of isolated build environments."""
 
-    def __init__(self) -> None:
-        self._path: Optional[str] = None
+    def __init__(self, isolated_env_class: Optional[Type[IsolatedEnv]] = None) -> None:
```

Comment:
```suggestion
    def __init__(self, isolated_env_class: Type[IsolatedEnv] = _DefaultIsolatedEnv) -> None:
```

why not?

Reply:
Because `_DefaultIsolatedEnv` is defined underneath `IsolatedEnvBuilder` and I didn't wanna swap them around in the draft.